.github/workflows/release-pablo-fe-staging.yml

```diff
@@ -0,0 +1,26 @@
+name: "Pablo Vercel Staging Deployment"
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.PABLO_PROD_PROJECT_ID }}
+on:
+  push:
+    tags:
+      - 'staging-fe-v[0-9]+\.[0-9]+-pablo-[0-9]'
+
+jobs:
+  Deploy-Staging:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Vercel CLI
+        run: npm install --global vercel@latest
+      - name: Import project setting
+        run: |
+          mkdir -p .vercel
+          echo '${{ secrets.PABLO_PROJ_JSON }}' >> .vercel/project.json
+      - name: Pull Vercel Environment Information
+        run: vercel pull --yes --environment=preview --token=${{ secrets.VERCEL_TOKEN }}
+      - name: Build Project Artifacts
+        run: vercel build --token=${{ secrets.VERCEL_TOKEN }}
```

Comment:
Can we create a backlog ticket to nixify this (don't want us to test and build using flow 1, and then build the deploy builds using 2)

Reply:
Created a task for this.